mesonbuild/interpreter.py

```diff
@@ -2263,6 +2264,42 @@ def _get_native() -> str:
             else:
                 return _get_native()
 
+    @permittedKwargs({'encoding', 'allow_absolute'})
+    @FeatureNew('meson.get_file_contents', '0.58.0')
+    def get_file_contents_method(
+            self, args: T.Sequence[str], kwargs: dict) -> str:
```

Comment:
that should be `kwags: T.Dict[str, T.Any]

Reply:
done